securedrop/manage.py

```diff
@@ -66,9 +66,13 @@ def add_journalist():  # pragma: no cover
     return _add_user()
 
 
+def get_username():  # pragma: no cover
```

Comment:
I agree with @dachary here - let's either add a test (better) or remove the no cover. We should be removing the no covers from most of the functions in this file (see #1982).